src/main/java/com/jagrosh/jmusicbot/settings/SettingsManager.java

```diff
@@ -31,14 +31,15 @@
  */
 public class SettingsManager implements GuildSettingsManager
 {
+    private final static String SETTINGS_FILE = "serversettings.json";
```

Comment:
Having this constant here is actually nice & fine I guess, albeit out of scope of this PR.